__tests__/append.spec.ts

```diff
@@ -10,17 +10,23 @@ describe('#write()', () => {
     memFs = create(createMemFs<MemFsEditorFile>());
   });
 
-  it('appends new content to new file', () => {
-    memFs.append('append.txt', 'b', { create: true });
+  it('appends new content to a new file', () => {
+    memFs.append('append.txt', 'b');
 
     expect(memFs.read('append.txt')).toBe('b');
   });
 
-  it('appends new content to file', () => {
+  it('skip creating new file', () => {
+    memFs.append('append.txt', 'b', { create: false });
+
+    expect(memFs.exists('append.txt')).toBe(false);
+  });
```

Comment:
This wasn't covered by a test, and it was also broken 😅

Reply:
It was not broken, append should not fail silently:

```
  it(`fails if file doesn't exist`, () => {
    expect(() => {
      memFs.append('append.txt', 'b', { create: false });
    }).toThrow();
  });
```